docs/qa/method.md

```diff
@@ -62,19 +62,35 @@ This section explains how the tests were carried out for reproducibility purpose
      desired proportion of nodes running each of the two configured versions.
 4. Follow steps 5-10 of the `README.md` to configure and start the 200 node testnet
     * WARNING: Do NOT forget to run `make terraform-destroy` as soon as you are done with the tests (see step 9)
-5. As a sanity check, connect to the Prometheus node's web interface and check the graph for the `COMETBFT_CONSENSUS_HEIGHT` metric.
-   All nodes should be increasing their heights.
-6. You now need to start the load runner that will produce transaction load
+5. As a sanity check, connect to the Prometheus node's web interface (port 9090)
```

Comment:
Prometheus is listed 2 in the requirements above (line 44).
Do they mean different things?